cmd/ipfs-cluster-service/main.go

```diff
@@ -213,6 +213,31 @@ configuration.
 				cfgMgr, cfgs := makeConfigs()
 				defer cfgMgr.Shutdown() // wait for saves
 
+				var alreadyInitialized bool
+				if _, err := os.Stat(configPath); !os.IsNotExist(err) {
+					alreadyInitialized = true
+				}
+
+				if alreadyInitialized {
+					// acquire lock for config folder
+					err := locker.lock()
+					checkErr("acquiring execution lock", err)
+					defer locker.tryUnlock()
+
+					if !c.Bool("force") {
+						if !yesNoPrompt("The peer's state will be removed from the load path.  Existing pins may be lost.\nConfiguration(service.json) will be overwritten.  Continue? [y/n]:") {
+							return nil
+						}
+					}
+
+					err = cfgMgr.LoadJSONFromFile(configPath)
+					checkErr("reading configuration", err)
+
+					err = cleanupState(cfgs.consensusCfg)
+					checkErr("Cleaning up consensus data", err)
+					logger.Warningf("the %s folder has been rotated. Starting with an empty state", cfgs.consensusCfg.GetDataFolder())
```

Comment:
Consolidate this warning in `cleanupState` perhaps, so the message is only in one place and not duplicated in the places where the function is called.

Reply:
If we do that the warning would show up even if cleanupState fails. Use a constant for warning message?